src/components/Popup.tsx

```diff
@@ -73,15 +76,18 @@ const Popup = ({ children, onClose, title, hookCoordinates, extraHeaderContent }
       style={hookOffset && { x: hookOffset.x, y: hookOffset.y - 15 }}
       animate={hookOffset && { ...fadeInOutScaleFast.animate, ...hookOffset }}
       exit={fadeInOutScaleFast.exit}
+      minWidth={minWidth}
       {...fastTransition}
     >
       {title && (
-        <Header hasExtraHeaderContent={!!extraHeaderContent}>
+        <Header>
           <h2>{title}</h2>
           {extraHeaderContent}
         </Header>
       )}
-      {children}
+      <Scrollbar translateContentSizeYToHolder isDynamic noScrollX>
+        {children}
+      </Scrollbar>
```

Comment:
This is what is causing the `Popup` to not close when clicking `ESC` (and probably all other accessibility issues)

Reply:
Fixed in https://github.com/alephium/desktop-wallet/pull/613